images/win/scripts/SoftwareReport/SoftwareReport.VStudio.psm1

```diff
@@ -0,0 +1,65 @@
+function Get-VSPackages
```

Comment:
Let's rename this helper to `SoftwareReport.VisualStudio.psm1`

Reply:
And first two functions too

Reply:
Done